test/fx/test_fx_split.py

```diff
@@ -30,3 +34,115 @@ def forward(self, x, y):
                 if n.op != "output":
                     self.assertIn("name", n.meta)
                     self.assertEqual(n.meta["name"], n.name)
+
+
+class TestSplitByTags(unittest.TestCase):
```

Comment:
This should be using the TestCase imported from torch.testing._internal.common_utils, not unittest